src/liballoc/boxed.rs

```diff
@@ -276,8 +284,11 @@ impl<T: Clone> Clone for Box<T> {
     /// let x = Box::new(5);
     /// let y = x.clone();
     /// ```
+    #[rustfmt_skip]
```

Comment:
No, it was reformatted, then I manually un-re-formatted part of it and added the attribute